setup.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+# ----------------------------------------------------------------------------
+# Copyright © 2022, kite-provider
+#
+# Licensed under the terms of the MIT license
+# ----------------------------------------------------------------------------
+"""
+kite-provider setup.
+"""
+import io
+from setuptools import find_packages
+from setuptools import setup
+
+# =============================================================================
+# Use Readme for long description
+# =============================================================================
+with io.open("README.md", encoding="utf-8") as f:
+    LONG_DESCRIPTION = f.read()
+
+setup(
+    # See: https://setuptools.readthedocs.io/en/latest/setuptools.html
+    name="kite-provider",
+    version="0.1.0",
+    author="",
```

Comment:
Lets put here `Spyder Project Contributors`:
```suggestion
    author="Spyder Project Contributors",
```